commands-yml/commands/element/find-element.yml

```diff
@@ -0,0 +1,84 @@
+---
+name: Find Element
+short_description: Click on an element
+description:
+  >
+     Located element will be returned as a MobileElement JSON object.
+
+     The locator strategy returns the first element it finds. #TODO: Let's make a document with the locator strategies that this links to
```

Comment:
we talk about locator strategies in the new getting started doc, could link to that?

Reply:
we could also put the list of standard locator strategies here, and mention that drivers may support additional strategies?

Reply:
or if this is the one-stop shop for all drivers, we should list them in a table here I guess

Reply:
Yeah I think we should have a full table listing all of the strategies and a description for them and the support levels.